cmd/synchronization/app/options.go

```diff
@@ -21,15 +21,17 @@ type Options struct {
 }
 
 type Providers struct {
-	ServiceCentralURL *url.URL // we use custom json marshalling to read it
-	DeployinatorURL   *url.URL
+	ServiceCentralURL              *url.URL // we use custom json marshalling to read it
+	DeployinatorURL                *url.URL
+	OpsgenieIntegrationsManagerURL *url.URL
 }
 
 // UnmarshalJSON unmarshals our untyped config file into a typed struct including URLs
 func (p *Providers) UnmarshalJSON(data []byte) error {
 	var rawProviders struct {
-		ServiceCentral string `json:"serviceCentral"`
-		Deployinator   string `json:"deployinator"`
+		ServiceCentral              string `json:"serviceCentral"`
+		Deployinator                string `json:"deployinator"`
+		OpsgenieIntegrationsManager string `json:"opsgenieIntegrationsManager"`
```

Comment:
Would be much more readable to name both json fields and struct fields like they are named in the `Providers` struct (camel case for JSON). I don't remember if we use `URL` or `Url` for Json fields, check other places.

Reply:
I'll add that to the list of follow up PRs, unless you want to block this PR on it.